src/item.cc

```diff
@@ -575,25 +575,34 @@ string item_context(const item_t& item, const string& desc)
     return empty_string;
 
   std::streamoff len = item.pos->end_pos - item.pos->beg_pos;
-  if (! (len > 0))
+  if (len <= 0)
     return empty_string;
 
   assert(len < 1024 * 1024);
 
   std::ostringstream out;
 
   if (item.pos->pathname.empty()) {
-    out << desc << _(" from streamed input:");
+    // FIXME: Find out what `desc` represents and update comment for translators
```

Comment:
@jwiegley Can you provide more information what values `desc` can hold? I assume the variable name is an abbreviation of `description`, yet that does not tell a translator much about how to best fit this piece into a sentence…